pkg/providers/tinkerbell/testdata/expected_results_cluster_tinkerbell_cp_stacked_etcd.yaml

```diff
@@ -180,21 +180,17 @@ spec:
               CONTENTS: |
                 datasource:
                   Ec2:
-                    metadata_urls:
-                    - http://1.2.3.4:50051
+                    metadata_urls: []
                     strict_id: false
-                manage_etc_hosts: localhost
                 system_info:
                   default_user:
-                    groups:
-                    - wheel
-                    - adm
                     name: tink
+                    groups: [wheel, adm]
+                    sudo: ["ALL=(ALL) NOPASSWD:ALL"]
                     shell: /bin/bash
-                    sudo:
-                    - ALL=(ALL) NOPASSWD:ALL
+                manage_etc_hosts: localhost
                 warnings:
-                  dsid_missing_source: false
+                  dsid_missing_source: off
```

Comment:
where are these changes coming from?

Reply:
They're generated. I regenerated the golden files and it found we clearly never generated them before. I assume we hand crafted them previously?